### PR TITLE
Addition of matrix in java

### PR DESCRIPTION
**Pull request by raksha009**
Issue No #1628

#### Checklist:
- [X] 4 space indentation.
- [X] Coding conventions are followed.
- [X] Input is taken dynamically.
- [X] Sample Input / Output is added at the end of file.
- [X] Logic Documentation (Comments).
- [X] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Addition of new code
- 

#### Languages Used:
- Java
- 

#### Files Added:
- Matrix_Addition.java
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by raksha009** [comment]
@asha15 maam, plz review my PR.

**Comment by raksha009** [comment]
@asha15 maam, i have made the changes. Plz review.

File: Matrix_Operations/Java/Matrix_Addition.java [hunk 1]
```diff
@@ -0,0 +1,109 @@
+// Java Program to calculation sum of 2 matrices (2D Array)
+import java.util.*;
+
+public class Matrix_Addition {
+    // Find sum of 2 Matrices and store it in matrix3 
+    public static void add_matrices(int[][] matrix1, int[][] matrix2) {
+        int index1, index2 = 0;
+        int[][] matrix3 = new int[matrix1.length][matrix1[0].length];
```
Comment (asha15) [inline Matrix_Operations/Java/Matrix_Addition.java]: keep a line space

File: Matrix_Operations/Java/Matrix_Addition.java [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+            System.out.println(" Enter elements of 2nd matrix ");
+            for (int i = 0 ; i < rows_first; i++) {
+                for (int j = 0 ; j< columns_first; j++) {
+                    matrix2[i][j] = s.nextInt();
+                }
+            }
+            add_matrices(matrix1, matrix2);
+        }
+    }
+}
```
Comment (asha15) [inline Matrix_Operations/Java/Matrix_Addition.java]: keep a line space

**Comment by raksha009** [comment]
done the changes @asha15 maam

**Comment by asha15** [comment]
could you squash the commits? then we are good to merge

**Comment by raksha009** [comment]
yes, I squash the commits right now.

**Comment by raksha009** [comment]
Squashed the commits @asha15  maam

**Pull request merged**